web/css/source/_theme.less

```diff
@@ -195,6 +226,7 @@ p,li,dt,dd,td,th,details,button,form,a,div,details {
     .content-container{
       padding-left: 15%;
       display: block;
+      celear:both;
```

Comment:
AH!!! That's way I couldn't get those elements to clear. Damn it, makes perfect sense now. I ended up adding a clearfix element, so no longer needed, but at the time, I was almost screaming at the screen "Magento, why don't you listen to me!" Turns out I was wrong this time.